src/objectTranslations/Engagement_Plan__c-fr.objectTranslation

```diff
@@ -2,35 +2,35 @@
 <CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
     <caseValues>
         <plural>false</plural>
-        <value><!-- Engagement Plan --></value>
+        <value>Plan d&apos;engagement</value>
     </caseValues>
     <caseValues>
         <plural>true</plural>
-        <value><!-- Engagement Plan --></value>
+        <value>Plans d&apos;engagement</value>
     </caseValues>
     <fields>
         <label>Compte</label>
         <name>Account__c</name>
-        <relationshipLabel>Plans d’engagement</relationshipLabel>
+        <relationshipLabel><!-- Engagement Plans --></relationshipLabel>
```

Comment:
The translations got reverted here and for the same "relationshipLabel"s below as well. Is this expected? @force2b

Reply:
Eagle Eyes!!! Great catch @GloryChoe! I confirm that this is missing in this PR and in another test one that generated using a slightly different technique. Looking into it this morning and will let you know what I find.

Reply:
@GloryChoe After reiewing, I can see that the Related List translations for the Account__c, Contact__c, Case__c, Campaign__c, Opportunity__c and RecurringDonation__c fields are not in original PR that this work is based on. You should be able to see that in this file from the parent `feature/ngohana/2019.07.localization` branch: https://github.com/SalesforceFoundation/Cumulus/blob/feature/ngohana/2019.07.localization/src/objectTranslations/Engagement_Plan__c-fr.objectTranslation

The result is that there the related list label translations applied are not in the scratch org this they don't exist in the metadata, which is why they're not in what gets pulled back from the scratch org.
<img width="818" alt="image" src="https://user-images.githubusercontent.com/801017/70718169-b9cf7900-1cbd-11ea-94be-17667349dc2a.png">

I searched through the PR and if I'm not mistaken this is the only object with missing Related List Label translations. I think the question here is why did the translation tool not include these.

Reply:
Hi @force2b - I've finished my review. All looks good except the reverted translations for Engagement Plans mentioned above. Thanks!

Reply:
@GloryChoe The EngagementPlan issue should now be resolved via https://github.com/SalesforceFoundation/Cumulus/pull/4878/commits/8a11b367d69dc9a0955b2becc3ffa0e57e7cb4b5